samples/Basic-TFRecorder-Usage.ipynb

```diff
@@ -26,17 +36,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 3,
    "metadata": {},
    "outputs": [],
    "source": [
     "input_file = pathlib.Path(os.getcwd())/'../tfrecorder/test_data/data.csv'\n",
-    "output_dir = './out'  # YOUR LOCAL OUTPUT PATH HERE"
+    "output_dir = 'D:\\Open Source\\Google\\out'  # YOUR LOCAL OUTPUT PATH HERE"
```

Comment:
Please change the `output_dir` value to something simpler, like `output`. The current path will not expand correctly in a Unix-based system.

Also at the end of the notebook, consider adding a cell to delete the generated output directory.

Reply:
done